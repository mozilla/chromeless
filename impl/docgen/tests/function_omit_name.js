/**
 * A module with a basic function definition in it,
 * but without explicit mention of the name of the function
 * given the heuristic name guessing stuff.
 */

/**
 * @function
 * a function which converts URLs to filenames
 */

let toFilename = exports.toFilename = function toFilename(url) {
}