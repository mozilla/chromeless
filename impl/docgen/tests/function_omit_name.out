{
  "desc": "A module with a basic function definition in it,\nbut without explicit mention of the name of the function\ngiven the heuristic name guessing stuff.", 
  "filename": "tests/function_omit_name.js", 
  "functions": [
    {
      "desc": "a function which converts URLs to filenames", 
      "name": "toFilename"
    }
  ], 
  "module": "function_omit_name"
}
